Utils/Tests/Inf-Test-1/Task1.cpp

```diff
@@ -1 +1,27 @@
-TBA
```

Comment:
@dejansvetoslavov Can you cherry-pick this commit/task ?
This is the only thing we need from this PR.

Reply:
@Justsvetoslavov done